css/blocks.css

```diff
@@ -1269,7 +1269,66 @@ h6.has-background {
 }
 
 .wp-block-image.aligncenter {
+  margin: 6rem auto;
```

Comment:
@GioAceto this seems to have increased the margin on center aligned images. Can we remove this style and see what it looks like?